.github/workflows/github-actions.yml

```diff
@@ -15,13 +15,27 @@
 # specific language governing permissions and limitations
 # under the License.
 
-sudo: false
-language: go
-go:
-  - 1.14.x
+name: Github-Actions
 
-script:
-  - make all
+on: [push, pull_request]
 
-env:
-  - GO111MODULE=on
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
@rahulbcn27 LGTM, Github Actions seems to be working. Can you raise a new PR to change runs-on to ubuntu-22.04 (see apache/cloudstack for example).

Reply:
https://github.com/rahulbcn27/cloudstack-cloudmonkey/pull/1